packages/react-server-dom-webpack/src/ReactFlightWebpackNodeLoader.js

```diff
@@ -210,10 +210,10 @@ async function parseExportNamesInto(
             addExportNames(names, node.declaration.id);
           }
         }
-        if (node.specificers) {
-          const specificers = node.specificers;
-          for (let j = 0; j < specificers.length; j++) {
-            addExportNames(names, specificers[j].exported);
+        if (node.specifiers) {
+          const specifiers = node.specifiers;
+          for (let j = 0; j < specifiers.length; j++) {
+            addExportNames(names, specifiers[j].exported);
```

Comment:
How did this ever work?

Reply:
Sorry, I didn't understand your question.

Reply:
How did the code that was there before– `node.specificers` –ever work?

Reply:
I'm confused, too.

Reply:
It probably didn’t. :) would be good to recreate a case where this codepath runs.

Reply:
I tagged Sebastian on the original PR.